app.js

```diff
@@ -6,6 +8,24 @@ const signUpForm = document.getElementById('sign-up');
 const signUpEmail = document.getElementById('sign-up-email');
 const signUpPassword = document.getElementById('sign-up-password');
 
-// Wire up sign in and sign up forms to supabase
-// Redirect to /other-page on successful auth
 // Redirect to /other-page when page loads if user is authenticated
+redirectIfLoggedIn();
+
+// Wire up sign in and sign up forms to supabase
+signUpForm.addEventListener('submit', async (e) => {
+    e.preventDefault();
```

Comment:
Nice work here another way you could have wrote this is like so but either way works just fine                        
   `const formData = new FormData(signInForm);
    const user = {
        email: formData.get('email'),
        password: formData.get('password')
    };
    await signInUser(user.email, user.password);`